src/unstract/sdk/index.py

```diff
@@ -388,14 +395,25 @@ def generate_file_id(
         index_key = {
             "tool_id": tool_id,
             "file_hash": file_hash,
-            "vector_db_config": ToolAdapter.get_adapter_config(self.tool, vector_db),
-            "embedding_config": ToolAdapter.get_adapter_config(self.tool, embedding),
-            "x2text_config": ToolAdapter.get_adapter_config(self.tool, x2text),
             # Typed and hashed as strings since the final hash is persisted
             # and this is required to be backward compatible
             "chunk_size": str(chunk_size),
             "chunk_overlap": str(chunk_overlap),
         }
+
+        if not self.is_public_call:
```

Comment:
Guess, this will eliminate adapter config combinations from the index_key. Ideally this was used to track any chnage in this combo in order to re-index. Since we are taking this out now, we might be losing out on that. Hope that will not cause a problem @tahierhussain

Reply:
Got it cleared.